### PR TITLE
[es] Update Discussion Channels

### PR DESCRIPTION
**Pull request by Graywolf9**
### Description

Update Discussion Channels for Spanish locale

### Motivation

Get in contact with more contributors

### Related issues and pull requests

Relates to https://github.com/mdn/content/pull/34382

### CODE REVIEW

File: files/es/mdn/community/contributing/translated_content/index.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ Hemos _congelado_ todo el contenido localizado (lo que significa que no aceptare
 
 ### Español (es)
 
-- Discusiones: [Matrix (canal #mdn-l10n-es)](https://chat.mozilla.org/#/room/#mdn-l10n-es:mozilla.org), [Telegram (canal MDN l10n ES)](https://t.me/+Dr6qKQCAepw4MjFj), [Discord (MDN Web Docs Community ,canal #spanish)](/discord)
+- Discusiones: [Matrix (`#mdn-l10n-es`)](https://chat.mozilla.org/#/room/#mdn-l10n-es:mozilla.org), [Telegram (`MDN l10n ES`)](https://t.me/+Dr6qKQCAepw4MjFj), [Discord (`#spanish`)](/discord)
 - Colaboradores actuales: [Graywolf9](https://github.com/Graywolf9), [JuanVqz](https://github.com/JuanVqz), [Jalkhov](https://github.com/Jalkhov), [marcelozarate](https://github.com/marcelozarate), [davbrito](https://github.com/davbrito), [Vallejoanderson](https://github.com/Vallejoanderson).
```
Comment (JuanVqz) [inline files/es/mdn/community/contributing/translated_content/index.md:51]: Maybe out of the scope but probably good time to remove some colaboradores.

**Review by JuanVqz — APPROVED** [review]

File: files/es/mdn/community/contributing/translated_content/index.md (hunk 1 above)
Comment (Graywolf9) [reply files/es/mdn/community/contributing/translated_content/index.md:51]: I was thinking about that, but maybe in another PR once the roles has been modified, not sure, what do you think?

Comment (JuanVqz) [reply files/es/mdn/community/contributing/translated_content/index.md:51]: yeah! It's fine as you did for now :+1:

File: files/zh-cn/mdn/community/contributing/translated_content/index.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ l10n:
 
 ### 西班牙语（es）
 
-- 讨论组：[Matrix (#mdn-l10n-es channel)](https://chat.mozilla.org/#/room/#mdn-l10n-es:mozilla.org)
+- 讨论组：[Matrix (`#mdn-l10n-es`)](https://chat.mozilla.org/#/room/#mdn-l10n-es:mozilla.org), [Telegram (`MDN l10n ES`)](https://t.me/+Dr6qKQCAepw4MjFj), [Discord (`#spanish`)](/discord)
```
Comment (yin1999) [inline files/zh-cn/mdn/community/contributing/translated_content/index.md:54]:
```suggestion
- 讨论组：[Matrix (`#mdn-l10n-es`)](https://chat.mozilla.org/#/room/#mdn-l10n-es:mozilla.org)、[Telegram (`MDN l10n ES`)](https://t.me/+Dr6qKQCAepw4MjFj)、[Discord (`#spanish`)](/discord)
```

File: files/zh-tw/mdn/community/contributing/translated_content/index.md [hunk 3]
```diff
@@ -51,7 +51,7 @@ l10n:
 
 ### 西班牙文（`es`）
 
-- 討論群組：[Matrix（`#mdn-l10n-es`）](https://chat.mozilla.org/#/room/#mdn-l10n-es:mozilla.org)、[Telegram（`MDN l10n ES`）](https://t.me/+Dr6qKQCAepw4MjFj)
+- 討論群組：[Matrix (`#mdn-l10n-es`)](https://chat.mozilla.org/#/room/#mdn-l10n-es:mozilla.org), [Telegram (`MDN l10n ES`)](https://t.me/+Dr6qKQCAepw4MjFj), [Discord (`#spanish`)](/discord)
```
Comment (yin1999) [inline files/zh-tw/mdn/community/contributing/translated_content/index.md:54]:
```suggestion
- 討論群組：[Matrix（`#mdn-l10n-es`）](https://chat.mozilla.org/#/room/#mdn-l10n-es:mozilla.org)、[Telegram（`MDN l10n ES`）](https://t.me/+Dr6qKQCAepw4MjFj)、[Discord（`#spanish`）](/discord)
```

**Review by yin1999 — COMMENTED** [review]
Chinese uses different punctuation marks to indicate parallelism and additional information. So let's do some updates :)

**Comment by Graywolf9** [comment]
Thank you so much for your comments @yin1999 !

**Review by yin1999 — APPROVED** [review]
LGTM. Thank you 👍

**Review by leon-win — APPROVED** [review]
Good for `ru` :+1:

**Review by cw118 — APPROVED** [review]
Looks good for l10n-fr, thank you!

**Review by mfuji09 — APPROVED** [review]
Sorry for too late.
LGTM for `ja`!

**Comment by Graywolf9** [comment]
Thank you so much for all your reviews!

**Pull request merged**